### ISSUE TITLE
nethserver-freepbx: allow to open SIPS from NethServer web GUI

### ISSUE THREAD

**Issue by Stell0** [NethServer/dev#5232]
From NethServer web interface is now possible to open Asterisk default WebRTC ports and IAX default ports. Is often needed to also open SIP Secure port (TCP 5061 and 10000-20000 UDP) 

At the moment opening those ports require creating a new configuration in config db

**Components**

nethserver-freepbx-0.0.1

**Comment by gsanchietti** [comment]
In nethserver-testing:
- nethserver-freepbx-0.0.1-1.88.g5d123f2.ns7.noarch.rpm

*Test case*
- Try to enable the new option
- Check the ports are open inside the firewall

This enhancement will be released with #5074 .

**Comment by pinkra** [comment]
Ok, enabling the new options the related ports are open inside the firewall.

```
+ACCEPT     udp  --  0.0.0.0/0            0.0.0.0/0            multiport dports 4569,5036 /* Enable IAX from red interfaces */
+ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:8089 /* Enable WebRTC from red interfaces */
```
